portal-knightsconfig.json

```diff
@@ -0,0 +1,438 @@
+        "Description":"Choose the password used to access your dedicated server by the Admin user group.",
+        "Keywords":"admin,password",
+        "FieldName":"adminscredentialspassword",
+        "InputType":"RandomPassword",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.admins.credentials.password",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"admin_password_please_change",
+        "EnumValues":{}
+    },
+    {
+        "DisplayName":"Admin Can Login",
+        "Category":"Server Settings",
+        "Description":"Defines whether Admins can log in.",
+        "Keywords":"admin,can,Login",
+        "FieldName":"adminscanLogin",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.admins.canLogin",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "True": "true",
+            "False": "false"
+        }
+    },
+    {
+        "DisplayName":"Admin Can Kick Players",
+        "Category":"Server Settings",
+        "Description":"Defines whether Admins can kick other players from the server.",
+        "Keywords":"admin,can,Kick,Players",
+        "FieldName":"adminscanKickPlayers",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.admins.canKickPlayers",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "True": "true",
+            "False": "false"
+        }
+    },
+    {
+        "DisplayName":"Admin Can Modify World",
+        "Category":"Server Settings",
+        "Description":"Defines whether Admins can mine/build blocks/props.",
+        "Keywords":"admin,can,Modify,World",
+        "FieldName":"adminscanModifyWorld",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.admins.canModifyWorld",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "True": "true",
+            "False": "false"
+        }
+    },
+    {
+        "DisplayName":"Admin Can Interact With Crafting Stations",
+        "Category":"Server Settings",
+        "Description":"Defines whether Admins can interact with crafting stations.",
+        "Keywords":"admin,can,Interact,With,Crafting,Stations",
+        "FieldName":"adminscanInteractWithCraftingStations",
+        "InputType":"checkbox",
+        "IsFlagArgument":false,
+        "ParamFieldName":"$.admins.canInteractWithCraftingStations",
+        "IncludeInCommandLine":false,
+        "DefaultValue":"true",
+        "EnumValues":{
+            "True": "true",
+            "False": "false"
+        }
+    },
+    {
+        "DisplayName":"Admin Can Interact With Chests",
+        "Category":"Server Settings",
+        "Description":"Defines whether Admins can interact with chests. Should be used together with canModifyWorld in order to also prevent mining of chests.",
```

Comment:
Rather than "with canModifyWorld", which the user does not see, perhaps better to say "with Admin Can Modify World enabled"?